external/ag-website-shared/src/utils/runNxGenerateExample.ts

```diff
@@ -0,0 +1,55 @@
+import { execSync } from 'child_process';
```

Comment:
This file will be resolved once I sort out https://github.com/ag-grid/ag-charts/pull/4995 - for this PR, it's just a side effect of the syncing